packages/bits/src/lib/sorter/sorter.component.ts

```diff
@@ -235,4 +257,13 @@ export class SorterComponent implements OnChanges, OnInit, OnDestroy, AfterViewI
     private onAppendToBodyChange(appendToBody: boolean): void {
         this.customContainer = appendToBody ? undefined : this.popupArea;
     }
+
+    private setKeyboardManagerServiceData(): void {
+        this.sorterKeyControlService.menuItems = this.menuPopup.menuItems;
+        this.sorterKeyControlService.overlay = this.overlay;
+    }
+
+    private initKeyboardManager() {
+        this.sorterKeyControlService.initKeyboardManager();
+    }
```

Comment:
It seems like we can consolidate these two methods into a single `initKeyboardService` method.